lib/fortnox/api/repositories/base.rb

```diff
@@ -55,19 +55,21 @@ def self.set_headers(headers = {}) # rubocop:disable Naming/AccessorMethodName
         def initialize(keys_filtered_on_save: [:url], token_store: :default)
           @keys_filtered_on_save = keys_filtered_on_save
           @token_store_name = token_store
```

Comment:
It's used for better error messages here:
```
        def expired?(token)
          decoded_token = JWT.decode token, nil, false
          decoded_token[0]['exp'] < (Time.now.to_i - TIME_MARGIN_FOR_ACCESS_TOKEN_RENEWAL)
        rescue JWT::DecodeError
          raise Exception, "Could not decode access token for token store \"#{@token_store_name}\""
        end

```

Reply:
Well, since you set that in the token store instance, for use as a Redis key, there should be no reason to remember it here as well. Expose it on the TokenStore and just do: `raise Exception, "Could not decode access token for token store \"#{@token_store.name}\""`?